controller/main.go

```diff
@@ -168,9 +171,15 @@ func main() {
 			ServiceChanged: c.SetBalancer,
 			PoolChanged:    c.SetPools,
 		},
-		ValidateConfig: validation,
-		EnableWebhook:  *enableWebhook,
-	})
+		ValidateConfig:     validation,
+		EnableWebhook:      *enableWebhook,
+		EnableCertRotation: *enableCertRotation,
+	}
+	if *enableWebhook && *enableCertRotation {
+		config.CertDir = *certDir
```

Comment:
Just forward the parameters to the config without the if. If they are not set, they'll be empty.
If certRotation is not enabled, they'll not be used.